pkgs/top-level/all-packages.nix

```diff
@@ -1768,7 +1768,8 @@ with pkgs;
 
   checkbashisms = callPackage ../development/tools/misc/checkbashisms { };
 
-  ckb = libsForQt5.callPackage ../tools/misc/ckb { };
+  ckb-next = libsForQt5.callPackage ../tools/misc/ckb-next { };
+  ckb = ckb-next; # alias retained for backwards compatibility
```

Comment:
Please move the alias to `pkgs/top-level/aliases.nix`.

(A bunch of aliases are marked as `TODO: move to aliases.nix` in this file.)